### ISSUE TITLE
Image location

### ISSUE THREAD

**Issue by ligi** [airgap-it/airgap-raspberry-apk-signer#1]
you reference one raspberry pi image in the README - where is it? Would really like to try it out - but I cannot find it.

**Comment by dcale** [comment]
we tried to release it but the image is 4GB big and github allows us only to push images up to 2GB. We are now investigating if there is maybe a way to get a smaller version...

**Comment by ligi** [comment]
Thanks for the info! Perhaps a solution is to provide a diff to the standard raspberry pi image - would be nicer anyway as it is really hard to trust a raw 4GB image ;-)

**Comment by dcale** [comment]
I agree, however if we go with a diff, it will make it more difficult for people to actually use it (additional step...).

**Comment by lukeisontheroad** [comment]
I will create a diff in the next day. As this solution is quite a niche solution, I assume we can expect some technical understanding.
